crates/utils/src/tracing/grpc.rs

```diff
@@ -129,6 +131,30 @@ pub fn store_ntx_builder_trace_fn<T>(request: &http::Request<T>) -> tracing::Spa
     add_network_attributes(span, request)
 }
 
+fn remote_prover_trace_fn<T>(request: &http::Request<T>) -> tracing::Span {
+    let method = request.uri().path().rsplit('/').next().unwrap_or("Unknown");
+    let span = if method == "Prove" {
+        rpc_span!("remote-prover.rpc", "Prove")
+    } else {
+        rpc_span!("remote-prover.rpc", "Unknown")
```

Comment:
This is probably wrong. I struggle to infer the correct names for these based on the proto files.

Reply:
I think its correct.

We could try automate this as part of the build.rs somehow because it does allow introspection of the services and methods.

The main issue is that the span name must be a `&'static str` which is why we use the match.

Maybe we are using the wrong approach, and the span shouldn't contain the method name but just have a `method` field which we could set. But we can revisit that in a separate PR.

Reply:
Lets keep what we have today and see how our observability experience goes. Rack up a bunch of changes to make once we have used Honeycomb in anger.

I added the Status endpoint for the worker span now.